python/foglamp/plugins/south/cc2650async/cc2650async.py

```diff
@@ -0,0 +1,354 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Module for Sensortag CC2650 'async' type plugin """
+
+import copy
+import datetime
+import uuid
+import json
+import asyncio
+from foglamp.plugins.south.common.sensortag_cc2650 import *
+from foglamp.services.south import exceptions
+from foglamp.common import logger
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_DEFAULT_CONFIG = {
+    'plugin': {
+         'description': 'Python module name of the plugin to load',
+         'type': 'string',
+         'default': 'cc2650poll'
+    },
+    'bluetoothAddress': {
+        'description': 'Bluetooth MAC address',
+        'type': 'string',
+        'default': 'B0:91:22:EA:79:04'
+    },
+    'connectionTimeout': {
+        'description': 'BLE Device timeout value in seconds',
+        'type': 'integer',
+        'default': '10'
+    }
+}
+
+_LOGGER = logger.setup(__name__, level=20)
+
+sensortag_characteristics = characteristics
+
+
+def plugin_info():
+    """ Returns information about the plugin.
+
+    Args:
+    Returns:
+        dict: plugin information
+    Raises:
+    """
+
+    return {
+        'name': 'TI SensorTag CC2650 Async plugin',
+        'version': '1.0',
```

Comment:
are we not following semantic versioning? 1.0.0

Reply:
Deferred till a proper versioing system is in place.